src/JwksClient.js

```diff
@@ -36,9 +42,21 @@ export class JwksClient {
   }
 
   getKeys(cb) {
-    this.logger(`Fetching keys from '${this.options.jwksUri}'`);
+    if (Array.isArray(this.options.jwks)) {
+      this.logger('Returning directly provided keyset.');
+      return cb(null, this.options.jwks);
+    }
+
+    if (fs.existsSync(this.options.jwks)) {
```

Comment:
`existsSync` and then `readFileSync` will be bit of a performance killer. You should do something like:

```js
this.logger(`Fetching keys from '${this.options.jwks}'`);
fs.readFile(this.options.jwks, (err, data) => {
  if (err) {
    request({ uri: this.options.jwksUri, ..., (err, res) => { ... }});
  }
  const keys = JSON.parse(rawData);
  return cb(null, keys);
  };
  console.log(data);
});
```